Package

```diff
@@ -3,6 +3,7 @@
 plugin_name=modbus
 plugin_type=south
 plugin_install_dirname=ModbusC
+plugin_package_name=fledge-${plugin_type}-${plugin_name}
```

Comment:
Why add this here, it seems to add something that its not needed and is error prone. Th packaging code already generates the name, so if it still does that there is scope for these to diverge and cause things to break again.

Reply:
This is an optional change. We can achieve FOGL-7513 even without it as packaging scripts already taking care of it. Even if we go with this change it does not impact on any of the area.